### DISCUSSION TITLE
Calendar issue with wrong time bar display

### DISCUSSION THREAD

**Discussion by r4m1k707** [YetiForceCompany/YetiForceCRM#16685] [How to]
Hello!
How to fix this problem. The calendar shows the wrong date. When Friday is marked it states the Saturday date. When Thursday is marked it states Friday. I found this in your gitstable demo and as well on mine.
![Screenshot_562](https://user-images.githubusercontent.com/48440882/203595236-5146ecb4-e1fe-4424-8d91-8b7367a09899.png)
![Screenshot_563](https://user-images.githubusercontent.com/48440882/203595270-032b8223-684c-4a22-bcd4-7e769784506f.png)

**Comment by rskrzypczak** [comment]
Check your time zone in user preferences and server configuration panel in CRM.
..also check your browser's time zone.
similar: https://github.com/YetiForceCompany/YetiForceCRM/issues/16440
